src/Umbraco.Web.UI.Client/src/common/directives/components/umbmediagrid.directive.js

```diff
@@ -188,6 +190,22 @@ Use this directive to generate a thumbnail grid of media items.
                     }
                 }
             }
+            
+            /**
+            * Returns wether a item should be selectable or not.
+            */
+            function getSelectableState(item) {
+
+                // check if item is a folder or image
+                if (item.isFolder === true) {
```

Comment:
Pretty ugly that we test for strings with exact value `"true"`. Are you sure this is correct? In src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/mediapicker/mediapicker.controller.js it is real booleans?

Reply:
I believe @nielslyngsoe done this the exact same way as the  `onlyImages` existing check in the same file